fastlane/lib/fastlane/actions/bundle_install.rb

```diff
@@ -40,7 +40,7 @@ def self.gemfile_exists?(params)
         possible_gemfiles.each do |gemfile|
           gemfile = File.absolute_path(gemfile)
           return true if File.exist?(gemfile)
-          Helper.log.info "Gemfile not found at: '#{gemfile}'"
+          UI.message("Gemfile not found at: '#{gemfile}'")
```

Comment:
Should this be an error or a warning ? I don't think it should be a standard message.